src/overview.js

```diff
@@ -0,0 +1,360 @@
+/**
+ * Overview board manager.
+ */
+function OverviewManager(config, logger, trelloClient, date) {
+  this.config = config;
+  this.logger = logger;
+  this.trelloClient = trelloClient;
+
+  this.overviewBoard = null;
+  this.boards = null;
+  this.projects = null;
```

Comment:
This should be initialized to `{}` or `new Map()`. (The latter may ease iteration and lookup).